### PR TITLE
871738: Added UG documentation for new TextArea component

### PR DESCRIPTION
**Pull request by arunkumarssf3997**
### Bug description
Added UG documentation for new TextArea component.

### Root cause
NA

### Solution description
Added UG documentation for new TextArea component.

### Reason for not identifying earlier
 * [ ] Guidelines not followed. If yes, provide which guideline is not followed.

 * [ ] Guidelines not given. If yes, provide which/who need to address.
    Tag label `update-guideline-coreteam` or `update-guideline-productteam`. 

 * [x] If any other reason, provide the details here. 
     
### Areas tested against this fix
Provide details about the areas or combinations that have been tested against this code changes.
* [ ]  Tested against feature matrix. [Feature matrix link](https://syncfusion.sharepoint.com/sites/EJ2ProductOwners/Shared%20Documents/Forms/AllItems.aspx?viewid=ae81c682%2D3d0f%2D462a%2Db8ec%2D7358748d489d&id=%2Fsites%2FEJ2ProductOwners%2FShared%20Documents%2FGeneral%2FFeature%20Matrix%20%2D%20Documents)
* [x]  NA

### Is it a breaking issue?
* [ ]  Yes, Tag `breaking-issue`. 
* [x]  NO
 
 If yes, provide the breaking commit details / MR here. 
 
### Action taken
NA

 Feature matrix document updated
* [ ]  Yes
* [ ]  NO
* [x]  NA
 
Automation details - Mark `Is Automated` field as (Yes, Manual, Not Applicable) in corresponding JIRA task once the bug is automated. 
* [ ] BUnit, share corresponding MR.
* [ ] E2E or Manual Automation using tester - Make sure all items are automated with priority before release which can be tracked in [automation dashboard](https://syncfusion.atlassian.net/secure/Dashboard.jspa?selectPageId=43396).
 
If the same issue is reproduced in ej2, what will you do?
* [ ]  Resolved. Provide MR link.
* [ ]  NO. Created task to track it. Share task link. 
* [x]  NA
 
 Is this common issue need to be addressed in the same component or on other components in our platform? 
* [ ]  Yes - Need to check in other components, tag `needs-attention-coreteam` 
* [x]  No
  
### Output screenshots
NA

### Blazor Checklist
Confirm whether this feature is ensured in both Blazor Server and WASM
* [x]  NA
* [ ]  Yes
* [ ]  NO

Is there any new API or existing API name change?
* [ ]  Yes. If yes, Provide API Review task link.
* [x]  NO
  
Is there any existing behavior change due to this code change?
* [ ]  Yes. Add `breaking-change` label.
* [x]  NO

Do the code changes cause any memory leak and performance issue? (Test only if you suspect that your code may cause problem)
* [ ]  Yes
* [x]  NO

### Reviewer Checklist
* [x]  All provided information are reviewed and ensured.

### CODE REVIEW

File: ej2-javascript/textarea/ts/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+@import '../../node_modules/@syncfusion/ej2-base/styles/material.css';
+@import '../../node_modules/@syncfusion/ej2-inputs/styles/material.css';
+
+{% endhighlight %}
+{% endtabs %}
+
+## Adding TextArea to the application
+
+Add the HTML Textarea element with `e-input` class into your `index.html`.
```
Comment (saravanangn) [inline ej2-javascript/textarea/ts/getting-started.md:81]: Why do we need to add 'e-input' class here?

File: ej2-javascript/textarea/ts/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+{% tabs %}
+{% highlight ts tabtitle="index.ts" %}
+{% include code-snippet/textarea/default-textarea-cs1/index.ts %}
```
Comment (saravanangn) [inline ej2-javascript/textarea/ts/getting-started.md:85]: Explain the ts file implementation in words like below.

https://ej2.syncfusion.com/documentation/rating/getting-started#adding-rating-to-the-application

File: ej2-javascript/textarea/ts/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+{% highlight html tabtitle="index.html" %}
+{% include code-snippet/textbox/default-textarea-cs1/index.html %}
+{% endhighlight %}
+{% highlight css tabtitle="styles.css" %}
+{% include code-snippet/textbox/default-textarea-cs1/index.css %}
+{% endhighlight %}
+{% endtabs %}
+        
+{% previewsample "page.domainurl/code-snippet/textarea/default-textarea-cs1" %}
+
```
Comment (saravanangn) [inline ej2-javascript/textarea/ts/getting-started.md:96]: Run the application topic missing, after that explain the Getting and setting values.

https://ej2.syncfusion.com/documentation/rating/getting-started#run-the-application

**Comment by arunkumarssf3997** [comment]
rebuild

**Review by PrinceOliver — APPROVED** [review]

**Pull request merged**